README.rst

```diff
@@ -0,0 +1,85 @@
+html-table-parser-python3
+=========================
+
+This module consists of just one small class.
+Its purpose is to parse HTML tables without help of external modules.
+Everything I use is part of python 3.
+You can install this module, or just copy the class located in *parse.py* into your own code.
+
+Getting Started
+---------------
+
+Installing
+~~~~~~~~~~
+
+Run pip install to this repo
+
+.. code:: bash
+
+ pip install git+git://github.com/schmijos/html-table-parser-python3.git
+
+Usage
+~~~~~
+
+.. code:: python
+
+ import urllib.request
+ from pprint import pprint
+ from html_table_parser import HTMLTableParser
+ 
+ target = 'http://www.twitter.com'
+
+ # get website content
+ req = urllib.request.Request(url=target)
+ f = urllib.request.urlopen(req)
+ xhtml = f.read().decode('utf-8')
+
+ # instantiate the parser and feed it
+ p = HTMLTableParser()
+ p.feed(xhtml)
+ pprint(p.tables)
```

Comment:
imo the benefit to put this on readme is actually to make user only read the source when he need to inspect the method or find the bug with the library. 

it also make people interested because people can see how simple the library is.